.gitignore

```diff
@@ -14,3 +14,134 @@ nickTest/oldsettings.html
 nickTest/oldteams.html
 nickTest/oldupload.html
 nickTest/oldvisualization.html
+
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+pip-wheel-metadata/
+share/python-wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
+htmlcov/
+.tox/
+.nox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+*.py,cover
+.hypothesis/
+.pytest_cache/
+
+# Translations
+*.mo
+*.pot
+
+# Django stuff:
```

Comment:
Most of this stuff is very good that I just haven't gotten around to yet but as of now I still want the database to have test data for local testing, yes it could be done 'more efficiently' by having a separate download for the data but this works for the scope of this project. 'local_settings.py' and '*.log' are useful tools when running locally and make it easier to review errors from another machine so I want them both in for ease of use.

Reply:
IMO having the db files on git is not great, but if you insist I won't* bar you from it. Regarding log files, you could just as easily throw those on somewhere, and in all honesty it would be easier than making a commit just to read it on another machine. On Gentoo we use `wgetpaste` for log files and the like. Much better than commiting something and pushing it :P.

*can't :D

Reply:
For a small project such as this that is still in active development, is there a better solution you would suggest? I fully agree that there are better ways to manage data large scale but in this case this has been a simple and effective solution.

Reply:
A better solution? Create a Python script that can create the file and fill it in with some sample data. It'll be both simple and effective. I *believe* I saw that we're currently deploying in development mode, which is fine, but when we get to the phase where we have a deployment mode, we can automatically run the script only when it's on development. Just a thought.